extensions/aproc/proc/ingest/drivers/impl/cosmoskymed.py

```diff
@@ -48,38 +47,45 @@ def supports(self, url: str) -> bool:
     def identify_assets(self, url: str) -> list[Asset]:
         assets = []
         if self.browse_path is not None:
+            # Prepare file to retrieve to proceed in local storage
+            browse_path = AccessManager.prepare(self.browse_path)
             thumbnail_path = Driver.output_folder + '/' + self.get_item_id(url) + '/thumbnail'
             os.makedirs(thumbnail_path, exist_ok=True)
             self.thumbnail_path = thumbnail_path + '/thumbnail.jpg'
-            geotiff_to_jpg(self.browse_path, 50, 50, self.thumbnail_path)
+            geotiff_to_jpg(browse_path, 50, 50, self.thumbnail_path)
             assets.append(Asset(href=self.thumbnail_path,
                                 roles=[Role.thumbnail.value], name=Role.thumbnail.value, type=MimeType.JPG.value,
-                                description=Role.thumbnail.value, size=get_file_size(self.thumbnail_path), asset_format=AssetFormat.jpg.value))
+                                description=Role.thumbnail.value, size=AccessManager.get_file_size(self.thumbnail_path), asset_format=AssetFormat.jpg.value))
+
             quicklook_path = Driver.output_folder + '/' + self.get_item_id(url) + '/quicklook'
             os.makedirs(quicklook_path, exist_ok=True)
             self.quicklook_path = quicklook_path + '/quicklook.jpg'
-            geotiff_to_jpg(self.browse_path, 250, 250, self.quicklook_path)
+            geotiff_to_jpg(browse_path, 250, 250, self.quicklook_path)
             assets.append(Asset(href=self.quicklook_path,
                                 roles=[Role.overview.value], name=Role.overview.value, type=MimeType.JPG.value,
-                                description=Role.overview.value, size=get_file_size(self.quicklook_path), asset_format=AssetFormat.jpg.value))
-        assets.append(Asset(href=self.tif_path, size=get_file_size(self.tif_path),
+                                description=Role.overview.value, size=AccessManager.get_file_size(self.quicklook_path), asset_format=AssetFormat.jpg.value))
+
+            if browse_path != self.browse_path:
+                os.remove(browse_path)
```

Comment:
The system should never delete any file. If it is really needed, then this must be documented and we must use the comment "!DELETE!".

https://docs.google.com/document/d/1UTm5JUZ8F9ptlcya9v88hzg-iPWxeuCCziAD_oycloo/edit?tab=t.0